addons/account_edi_ubl/models/account_edi_format.py

```diff
@@ -257,10 +258,3 @@ def _post_invoice_edi(self, invoices):
             attachment = self._export_ubl(invoice)
             res[invoice] = {'attachment': attachment}
         return res
-
-    def _is_embedding_to_invoice_pdf_needed(self):
-        # OVERRIDE
-        self.ensure_one()
-        if self.code != 'ubl_2_1':
-            return super()._is_embedding_to_invoice_pdf_needed()
-        return False  # ubl must not be embedded to PDF.
```

Comment:
I don't understand how such diff is related to l10n_no

Reply:
It's not really. What I'm trying here is to uniformise this kind of code for all ubl formats. Since it does the same thing, it's easier to read if the code is the same.

I did this in a different commit though.